syntax = "proto3";

import "google/protobuf/any.proto";

message ServiceRequest {
    int32 request_id = 1;
    string serviceName = 3;
    google.protobuf.Any message = 4;
}

message ServiceResponse {
    int32 request_id = 1;
    string serviceName = 3;
    google.protobuf.Any message = 4;
}

message AddRequest {
    int32 numA = 1;
    int32 numB = 2;
}


message AddResponse {
    int32 result = 1;
}

message SimpleString {
    string aString = 1;
}
